app/src/pages/playground/Playground.tsx

```diff
@@ -50,35 +60,132 @@ function AddPromptButton() {
   const addInstance = usePlaygroundContext((state) => state.addInstance);
   const numInstances = usePlaygroundContext((state) => state.instances.length);
   return (
-    <Button
-      variant="default"
-      size="compact"
-      aria-label="add prompt"
-      icon={<Icon svg={<Icons.PlusCircleOutline />} />}
-      disabled={numInstances >= NUM_MAX_PLAYGROUND_INSTANCES}
-      onClick={() => {
-        addInstance();
+    <div
+      onClick={(e) => {
+        // Stop propagation to prevent the accordion from closing
+        e.stopPropagation();
       }}
     >
-      Prompt
-    </Button>
+      <Button
+        variant="default"
+        size="compact"
+        aria-label="add prompt"
+        icon={<Icon svg={<Icons.PlusCircleOutline />} />}
+        disabled={numInstances >= NUM_MAX_PLAYGROUND_INSTANCES}
+        onClick={() => {
+          addInstance();
+        }}
+      >
+        Prompt
+      </Button>
+    </div>
```

Comment:
<img width="1368" alt="image" src="https://github.com/user-attachments/assets/758ae472-6ef8-4685-a2cf-65bb9977bb57">

I'm not seeing this button

Reply:
Yeah you need to bump your deps

Reply:
I did. Weird, let me nuke node modules